modules/imaging_browser/php/NDB_Form_imaging_browser.class.inc

```diff
@@ -153,7 +153,7 @@ class NDB_Form_Imaging_Browser extends NDB_Form
             WHERE SessionID=:SID AND (AcquisitionProtocolID IS NULL 
             OR AcquisitionProtocolID not in (1, 2, 3, 52)) 
             AND PendingStaging=0 $extra_where_string 
-            ORDER BY files.OutputType, sel.Value DESC, AcquisitionProtocolID",
```

Comment:
maybe ORDER BY files.OutputType, files.File would be better? The order would still be stable, but they'd be grouped by output type..